elliott/elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -56,9 +65,100 @@ async def attach_cve_flaws_cli(
+
+        if output == 'json':
+            click.echo(json.dumps(release_notes.model_dump(mode='json'), indent=4))
+        else:
+            click.echo(YAML.dump(release_notes.model_dump(mode='python'), sys.stdout))
+
+    elif runtime.build_system == 'brew':
+        await handle_brew_cve_flaws(
+            runtime,
+            into_default_advisories,
+            advisory_id,
+            default_advisory_type,
+            noop,
+        )
+
+
+async def handle_konflux_cve_flaws(runtime: Runtime, default_advisory_type: str):
+    """
+    Handle attaching CVE flaws in a Konflux environment.
+    """
+
+    # Read shipment block from the assembly config
+    assembly_group_config = assembly_config_struct(runtime.get_releases_config(), runtime.assembly, "group", {})
+    shipment = assembly_group_config.get("shipment", {})
+    advisories = shipment.get("advisories", [])
+    if not advisories:
+        LOGGER.info("No advisories found in the shipment block, exiting.")
+        sys.exit(0)
+
+    # Validate default advisory type
+    kinds = [advisory.get("kind") for advisory in advisories]
+    if default_advisory_type not in kinds:
+        raise click.UsageError(
+            f"Default advisory type '{default_advisory_type}' not found in shipment advisories: {advisories.keys()}"
+        )
+
+    # Get the shipment configs from the merge request URL
+    mr_url = shipment.get("url")
+    if not mr_url:
+        raise click.UsageError("Shipment block does not contain a 'url' field for the merge request")
+    LOGGER.info(f"Fetching shipment configs from merge request: {mr_url}")
+
+    # Fetch the shipment configs from the merge request
+    release_notes = get_release_notes_from_mr(mr_url, default_advisory_type)
+
+    # Fetch the bug IDs from the release notes
+    bug_ids = [issue.id for issue in release_notes.issues.fixed] if release_notes.issues else []
+
+    if not bug_ids:
+        LOGGER.info("No fixed issues found in the release notes, exiting.")
+        sys.exit(0)
+
+    # Get the bug trackers
+    tracker_bugs = get_attached_trackers(bug_ids, runtime.get_bug_tracker('jira'))
+    tracker_flaws, flaw_bugs = get_flaws(runtime.get_bug_tracker('bugzilla'), tracker_bugs)
+
+    if flaw_bugs:
+        LOGGER.info(f"Found {len(flaw_bugs)} flaw bugs, updating release notes.")
+
+        # Turn the advisory type into an RHSA
+        release_notes.type = 'RHSA'
+
+        # Add the CVE component mapping to the cve field
+        cve_component_mapping = get_cve_component_mapping(flaw_bugs, tracker_bugs, tracker_flaws)
+        for cve_id, components in cve_component_mapping.items():
+            for component in components:
+                release_notes.cves.append(CveAssociation(key=cve_id, component=component))
+
+        # Attach the flaw bugs to the release notes issues
+        release_notes.issues.fixed.extend([{'id': bug.id, 'source': "issues.redhat.com"} for bug in flaw_bugs])
```

Comment:
```suggestion
        release_notes.issues.fixed.extend([{'id': bug.id, 'source': "bugzilla.redhat.com"} for bug in flaw_bugs])
```

Reply:
i see this thread as resolved .. but don't see the change in?

Reply:
Done with 905276eea392acef22dc14279f3a6499c1f734a5